src/pyJiraCli/ret.py

```diff
@@ -56,6 +56,7 @@ class Ret(IntEnum):
     RET_ERROR_MISSING_DATATYPE       = 11
     RET_ERROR_CREATING_TICKET_FAILED = 12
     RET_ERROR_INVALID_SEARCH         = 13
+    RET_ERROR_PARSE_ARGUMENTS        = 14
 
```

Comment:
if a new error is added, there should be a related error message in the printer class in RETURN_MSG dict

Reply:
Why are the messages in the printer class and not here?

Reply:
It was not possible because of the order of declaration.
Since all py-files in this project follow the order 
- Imports
- Vars
- Classes
- Functions
This means the ret class has to be declared after the RetMsg dict if they were in the same file
The only option would be to put the return messages in a separate file and not in the printer.py file